### DISCUSSION TITLE
Problem with Blog Format on PC

### DISCUSSION THREAD

**Discussion by spaceroam** [microfeed/microfeed#78] [Q&A] [answered]
Hi, I just discovered that the blog formatting appears to be messed up on computers, while it still looks fine on my iPhone. There is no spacing between lines now; everything appears squished together. Seems like all the formatting elements have disappeared. After discovering this issue, I updated to the latest version, but the problem still persists. I have been enjoying using Microfeed and I hope you can look into fixing this issue. Thank you!
![image](https://github.com/microfeed/microfeed/assets/69491108/90b3a42f-fd80-401f-9362-30f5262b5885)

**Comment by wenbinf** [comment]
Have you changed any code in the code editor?

A good place to start troubleshooting styling issues is in the code editor:
{your_domain_name}/admin/settings/code-editor/?type=themes&theme=custom#webHeader

<img width="1284" alt="Screenshot 2023-06-03 at 5 09 27 PM" src="https://github.com/microfeed/microfeed/assets/1719237/7ae8fdc2-c981-4f47-b649-1ae72e1cf775">

Once you learned how to update code there, you can customize your website's UI.

**Comment by spaceroam** [comment]
No, I haven't made any changes. This issue is not limited to just one blog; it has affected several of my blogs as well as those of my friends. Also, I initially found the default UI to be just fine for my needs. So it would be great if the original appearance could be restored. If that's not possible, then I will proceed with customizing the UI. Thank you anyway!

**Comment by spaceroam** [comment]
The current default line-height is 1.75, but there is clearly no spacing between lines when viewing on a computer. When I change this value, the line spacing changes on my phone, but nothing changes on the computer.

**Reply by wenbinf** [reply]
what browser and operating system do you use? 

can you share your public url here?

with these info, people who are good at css may be able to help you troubleshoot a bit :)

**Reply by spaceroam** [reply]
The issue occurs on Chrome, on both windows 10 and 11. I just tried Edge, and it displays normally. Thank you!

**Reply by wenbinf** [reply, marked as answer]
In theory, Edge and Chrome should behave the same, as they are based on the same rendering engine.

My best guess is that you may install some extensions on Chrome that may interfere the styling. 

Anyways, i'll take a look once i get access to a computer later today (using my phone now).

Meanwhile, you can inspect styling using Chome's dev tool and tweak styling there. https://developer.chrome.com/docs/devtools/css/

**Reply by spaceroam** [reply]
You're amazing! I tested some extensions and it turns out that one of them, which was updated today, was causing the issue! Once I disabled that extension, everything went back to normal. Thank you so much for your help in troubleshooting the problem. I really appreciate it!
